src/frontend/src/features/neighborhoods/types.ts

```diff
@@ -0,0 +1,55 @@
+import type { FeatureCollection, MultiPolygon, Point } from "geojson";
+
+interface NeighborhoodProperties {
+    crime_percentile: number;
```

Comment:
I'd double check the data types of these fields against the columns in the DB or the data models on the Django side, because it looks like at least the `crime_percentile` field is nullable- there might be more